bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1951,6 +1953,56 @@ public int runCmd(CommandLine cmdLine) throws Exception {
         }
     }
 
+    /**
+     * Migrate the specified ledger data on some bookie to other bookies.
+     * */
+    class ReplicasMigrationCmd extends MyCommand {
+
+        ReplicasMigrationCmd() {
+            super(CMD_REPLICASMIGRATION);
+            opts.addOption("b", "bookieIds", true,
+                    "Bookies corresponding to the migrated replica");
+            opts.addOption("l", "ledgerIds", true,
+                    "The ledgerIds corresponding to the migrated replica");
+            opts.addOption("r", "readOnly", true,
```

Comment:
1. required = false?
2. We'd better set the default readOnly flag to `false`. If the admin uses this command to decommission some bookies, which configured placement policy, it may lead to writing new entries failing due to not meeting placement policy.

Reply:
@hanggc0276
The readOnly default value has been set to false here:
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/19296967/190057439-77a31577-67a5-4d15-b5d7-c12bb3c354c7.png">

Reply:
OK, I will  fix!
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/19296967/190059205-2ff05bfb-3a73-4c0d-aa32-d3b80d43d3d3.png">

Reply:
Fixed. 
PTAL,thanks!  @hangc0276